explainer.md

```diff
@@ -17,40 +17,33 @@ The two drafts apply to both TLS server certificates, where the server authentic
+However, this extension's size is impractical for some applications. Existing PKIs may have many CAs, and existing CAs may have long X.509 names. As of August 2023, the Mozilla CA Certificate Program {{MOZILLA-ROOTS}} contained 144 CAs, with an average name length of around 100 bytes. While applications could mitigate this with smaller PKIs and transitioning to newer CAs with shorter names, this is incompatible with existing PKIs.
 
-* For a new CA to be usable by subscribers, it must be trusted by all relying parties. This is particularly challenging for older, unupdatable relying parties. Existing CAs face similar challenges when rotating or deploying new keys.
+Without a negotiation mechanism, the subscriber must instead obtain a single certificate which simultaneously satisfies all relying parties. This is a challenge for subscribers when relying parties are diverse. PKI evolution naturally leads to relying party diversity, so this translates to analogous challenges for CAs and relying parties:
 
-* When a relying party must update its policies to meet new security requirements, it must choose between compromising on user security or imposing a significant burden on subscribers that still support older relying parties.
-
-The two drafts aim to remove this constraint, by enabling a *multi-certificate deployment model*. Subscribers are instead provisioned with multiple certificates and automatically select the correct one to use with each relying party. This allows a single subscriber to use different certificates for different relying parties, including older and newer ones.
-
-There are three parts to understanding this proposal:
+* For a new CA to be usable by subscribers, all relying parties must trust it. This is particularly challenging for older, un-updatable relying parties. Existing CAs face similar challenges when rotating or deploying new keys.
 
-1. The multi-certificate model itself
-2. A TLS extension for relying parties to succinctly communicate trusted CAs
-3. An [ACME](https://www.rfc-editor.org/rfc/rfc8555.html) extension for provisioning multiple certificates
+* When a relying party must update its policies to meet new security requirements, it must choose between compromising on user security or imposing a significant burden on subscribers that still support older relying parties.
 
-Together, these aim to support a more flexible and robust Public Key Infrastructure (PKI).
+These challenges lead to a conflict between security and availability. To address this, the two proposals extend the `certificate_authorities` extension to reduce the size cost, supporting flexible and robust PKIs for more applications. The proposals also define a supporting [ACME](https://www.rfc-editor.org/rfc/rfc8555.html) extension to help subscribers provision multiple certificates.
```

Comment:
They don't define an ACME extension. The use an existing one. 

The proposals also define a new media type for certificate paths with identifying information, and how this can be used in conjunction with ACME<blah blah> to help subscribers provision and use multiple certificates.

Reply:
Defining a new media type is defining an extension. We are extending ACME. ACME doesn't have a concrete notion of extensions the way that TLS does.

Reply:
Also "ACME extension" comes from the existing text. :P